src/main/java/net/minecraftforge/common/extensions/IForgePlayer.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.common.extensions;
+
+import java.util.Optional;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.common.ForgeMod;
+
+public interface IForgePlayer {
```

Comment:
Formatting: braces on newlines.
```suggestion
public interface IForgePlayer
{
```